tutorials/power_distribution/api.h

```diff
@@ -0,0 +1,26 @@
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+
+gpio_t DASHBOARD_POWER = PB0;
+gpio_t SERVICE_SECTION_POWER = PB1;
+gpio_t THROTTLE_POWER = PB2;
+
+/*
+ * Power on the given device
+ */
+void power_device(gpio_t pin);
+
+/*
+ * Power on all devices
+ */
+void power_all();
```

Comment:
I think this is fine, but usually if a function doesn't take arguments, I specify `void`:

```c
void power_all(void);
```